docs/flamework/additional-modules/components/extra.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Extra
```

Comment:
Maybe this file should be called something like `Configuration` and include other configuration options?

`attributes`, `defaults`, `instanceGuard` and `refreshAttributes` are all undocumented at the moment.

Reply:
Hey, I agree entirely with the name suggestion and implemented that.

I've also added `refreshAttributes`, `defaults`, & `instanceGuard` to the page. 

I'm not 100% sure if the `instanceGuard`'s example is great because I've never used them and could only find one usage of them while searching through Discord.

I was also unable to find any usages of `attributes` in Discord. I'm assuming it can be used to specify Attributes, an alternative to providing the Component an interface containing them, but I wasn't 100% sure so I didn't include it.

Reply:
`attributes` allows you to specify custom type guards for attributes defined in your component's attribute interface. For example, you might only want constrained numbers in which case you could use `@rbxts/t` and `t.numberConstrained(min, max)` or specify your own function.